app/models.py

```diff
@@ -1039,6 +1039,10 @@ def validate(self, enforce_required=True, required_fields=None):
         ):
             errs['niceToHaveRequirements'] = 'answer_required'
 
+        if max_day_rate and 'dayRate' not in errs:
+            if float(self.data['dayRate']) > float(max_day_rate):
+                errs['dayRate'] = 'max_less_than_min'
```

Comment:
This key name seems confusing here.
Is there a benefit in re-using this key rather than having something new and more accurately descriptive? (e.g. `over_max_value`)